templates/workloads/common/cf/env-controller.yml

```diff
@@ -6,6 +6,8 @@ EnvControllerAction:
     EnvStack: !Sub '${AppName}-${EnvName}'
     Parameters: 
       - 'ALBWorkloads'
+    # We need to force trigger this lambda function on all deployments, so we give it a random ID as input on all event types.
+    UpdateID: {{ randomUUID }}
```

Comment:
What was the reason again we did this? 

It was so that when a new output is added to the env stack we can pick it up right?

Reply:
yeah